plotter/simple_plot.py

```diff
@@ -552,6 +568,46 @@ def getCanvas():
     c.SetBorderSize(0);
     return c
 
+def drawOneProfile(profiles,var,plotdir):
```

Comment:
this function is added to simple_plots to draw the cluster number isolation vs distance from center